Makefile

```diff
@@ -34,20 +34,20 @@ endif
 # Run e2e tests
 .PHONY: e2e-test
 e2e-test: cli-fast manager-fast
-	./hack/run-e2e-tests.sh
+	TEST_TO_RUN=$(TEST) ./hack/run-e2e-tests.sh
```

Comment:
`TEST_TO_RUN` sounds clunky to me. How about `TEST_ONLY` for good old time's sake? 😊

Reply:
I'm fine with that, especially as it's an internal variable only - it's not exposed anywhere.